awx/main/dispatch/worker/callback.py

```diff
@@ -142,7 +142,8 @@ def flush(self, force=False):
                             logger.exception('Database Error Saving Job Event')
                 duration_to_save = time.perf_counter() - duration_to_save
                 for e in events:
-                    emit_event_detail(e)
+                    if not e.event_data.get('skip_websocket_message', False):
```

Comment:
It's not completely unreasonable that we might change the `.get` to a `.pop` here and that'll remove the key from the API. It is useful for debugging purposes, but I don't know that we want it for production.

Reply:
I'm pro change to pop.